recipes/audiofile/all/conanfile.py

```diff
@@ -11,30 +12,42 @@
 class AudiofileConan(ConanFile):
     name = "audiofile"
     description = "A simple C++11 library for reading and writing audio files."
-    topics = ("audiofile", "audio", "file-format", "wav", "aif")
     license = "MIT"
-    homepage = "https://github.com/adamstark/AudioFile"
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/adamstark/AudioFile"
+    topics = ("audio", "file-format", "wav", "aif", "header-only")
+
+    package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
+    @property
+    def _min_cppstd(self):
+        # v1.1.1 uses is_signed_v
+        return 11 if Version(self.version) < "1.1.1" else 17
+
     def configure(self):
         if Version(self.version) < "1.1.0":
             self.license = "GPL-3.0-or-later"
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
     def package_id(self):
         self.info.clear()
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            check_min_cppstd(self, 11)
-
-    def layout(self):
-        basic_layout(self, src_folder="src")
+            check_min_cppstd(self, self._min_cppstd)
+        if (
```

Comment:
Please, follow the template, so we can check more compilers: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L46